R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -72,6 +72,20 @@ test_that("repeatedly starting and stopping SparkSession", {
   }
 })
 
+test_that("Default warehouse dir should be set to tempdir", {
+  sparkR.session.stop()
+  sparkR.session(enableHiveSupport = FALSE)
+
+  # Create a temporary table
+  sql("CREATE TABLE people_warehouse_test")
+  # spark-warehouse should be written only tempdir() and not current working directory
+  res <- list.files(path = ".", pattern = ".*spark-warehouse.*",
```

Comment:
Well - given that this PR is only changing the warehouse dir, I think its only fair to test for that.  Or in other words, adding such a test would fail now because of `derby.log` etc. (per our JIRA discussion) ?